examples/mobile/UIComponents/components/controls/floatingactions/on-listview.html

```diff
@@ -35,19 +37,11 @@ <h1>Floating Actions</h1>
 				<li class="ui-li-static">List Item 20</li>
 			</ul>
 			<div class="ui-floatingactions">
-				<button data-icon="floating-add"
-					data-focus-container-context="true"
-					data-focus-context=".ui-page-active"
-					data-focus-left=".ui-floatingactions"
-				/>
-				<button data-icon="floating-search"
-					data-focus-container-context="true"
-					data-focus-context=".ui-page-active"
-					data-focus-right=".ui-floatingactions"
-				/>
+				<button data-icon="floating-add" data-focus-container-context="true" data-focus-context=".ui-page-active" data-focus-left=".ui-floatingactions" />
```

Comment:
In my opinion this change doesn't look better. It was more human readable before.